src/Microsoft.IdentityModel.Tokens/JsonClaimValueTypes.cs

```diff
@@ -0,0 +1,53 @@
+﻿//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Constants for Json Web tokens.
+    /// </summary>
+    public static class JsonClaimValueTypes
```

Comment:
This type is in M.IM.JsonWebTokens, we shouldn't have two public types with the same name.

Reply:
JsonClaimValueTypes is used by GetClaimValueUsingValueType , due to circular dependency problem I had to create a new one as System.IdentityModel.Tokens.Jwt also has one :)

Reply:
I recognize the problem, but I agree with Brent here. I'm not sure this is how we want to solve this.

Reply:
Defining `JsonClaimValueTypes `consts locally in `TokenUtilities `to solve the problem at this moment.